<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionGroupes</class>
 <widget class="QDialog" name="GestionGroupes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>351</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de Mots de passe - Gestion des groupes</string>
  </property>
  <widget class="QLabel" name="titrePage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gestion des Groupes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QPushButton" name="supprimer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer le groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gererMotsDePasse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Gérer les mots de passe
du groupe</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listeGroupes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="titreSelection">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Sélection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="creer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Créer un groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier le groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gererUtilisateurs">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Gérer les utilisateurs
du groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="actualiser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QLabel" name="labelNom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom complet :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="annulerGroupe">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="titreCreerModifier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Créer|Modifier un groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validerGroupe">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page3">
    <widget class="QLabel" name="titreGererUtilisateurs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gérer les utilisateurs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validerUtilisateurs">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enleverUtilisateur">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUtilisateursGroupe">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Utilisateurs du groupe</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUtilisateurs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Liste des utilisateurs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annulerUtilisateurs">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>260</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterUtilisateur">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="utilisateurs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="utilisateursGroupe">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelGroupe1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Groupe</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page4">
    <widget class="QLabel" name="titreGererMotsDePasse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gérer les mots de passe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validerMotsDePasse">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enleverMotDePasse">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMotsDePasseGroupe">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mots de passe du groupe</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMotsDePasse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Liste des mots de passe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annulerMotsDePasse">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>260</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterMotDePasse">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="motsDePasse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="motsDePasseGroupe">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelGroupe2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Groupe</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>titrePage</zorder>
 </widget>
 <tabstops>
  <tabstop>creer</tabstop>
  <tabstop>listeGroupes</tabstop>
  <tabstop>actualiser</tabstop>
  <tabstop>gererUtilisateurs</tabstop>
  <tabstop>gererMotsDePasse</tabstop>
  <tabstop>modifier</tabstop>
  <tabstop>supprimer</tabstop>
  <tabstop>nom</tabstop>
  <tabstop>validerGroupe</tabstop>
  <tabstop>annulerGroupe</tabstop>
  <tabstop>utilisateurs</tabstop>
  <tabstop>utilisateursGroupe</tabstop>
  <tabstop>ajouterUtilisateur</tabstop>
  <tabstop>enleverUtilisateur</tabstop>
  <tabstop>validerUtilisateurs</tabstop>
  <tabstop>annulerUtilisateurs</tabstop>
  <tabstop>motsDePasse</tabstop>
  <tabstop>motsDePasseGroupe</tabstop>
  <tabstop>ajouterMotDePasse</tabstop>
  <tabstop>enleverMotDePasse</tabstop>
  <tabstop>validerMotsDePasse</tabstop>
  <tabstop>annulerMotsDePasse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
